packages/expo-image/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### 💡 Others
 
+## 1.0.0-beta.6
+
+- Added new prop `placeholderContentFit` to specify custom content fit on the placeholder. ([#21096](https://github.com/expo/expo/pull/21096) by [@magrinj](https://github.com/magrinj))
```

Comment:
Could you move it a few lines up, to `Unpublished / New features` section? Our publish script replaces the "Unpublished" with the version that is being published and removes other empty sections.

Reply:
@tsapeta Yes of course, I didn't take attention at the top of the file sorry 😅